src/assets/i18n/cs.json5

```diff
@@ -460,143 +522,469 @@
-  // "admin.access-control.groups.form.tooltip.editGroup.addEpeople": "To add or remove an EPerson to/from this group, either click the 'Browse All' button or use the search bar below to search for users (use the dropdown to the left of the search bar to choose whether to search by metadata or by email). Then click the plus icon for each user you wish to add in the list below, or the trash can icon for each user you wish to remove. The list below may have several pages: use the page controls below the list to navigate to the next pages. Once you are ready, save your changes by clicking the 'Save' button in the top section.",
-  "admin.access-control.groups.form.tooltip.editGroup.addEpeople": "Chcete-li přidat nebo odebrat EPersona do/z této skupiny, klikněte na tlačítko 'Procházet vše', nebo použijte vyhledávací lištu níže pro vyhledávání uživatelů (pomocí rozbalovacího seznamu vlevo na liště vyhledávání vyberte, zda chcete vyhledávat podle metadat, nebo e-mailem). Poté klikněte na ikonu plus pro každého uživatele, kterého si přejete přidat do seznamu níže, nebo na ikonu koše pro každého uživatele, kterého si přejete odstranit. Seznam níže může mít několik stránek: pomocí ovládacích prvků stránky pod seznamem přejděte na další stránky. Jakmile budete připraveni, uložte své změny kliknutím na tlačítko 'Uložit' v horní části.",
-  // "admin.access-control.groups.form.tooltip.editGroup.addSubgroups": "To add or remove a Subgroup to/from this group, either click the 'Browse All' button or use the search bar below to search for users. Then click the plus icon for each user you wish to add in the list below, or the trash can icon for each user you wish to remove. The list below may have several pages: use the page controls below the list to navigate to the next pages. Once you are ready, save your changes by clicking the 'Save' button in the top section.",
-  "admin.access-control.groups.form.tooltip.editGroup.addSubgroups": "Chcete-li přidat nebo odebrat podskupinu do/z této skupiny, klikněte na tlačítko 'Procházet vše', nebo použijte vyhledávací lištu níže pro vyhledávání uživatelů. Poté klikněte na ikonu plus pro každého uživatele, kterého si přejete přidat v seznamu níže, nebo na ikonu koše pro každého uživatele, kterého si přejete odstranit. Seznam níže může mít několik stránek: použijte ovládací prvky stránky pod seznamem pro navigaci na další stránky. Jakmile budete připraveni, uložte své změny kliknutím na tlačítko 'Uložit' v horní části.",
+  "admin.access-control.groups.form.tooltip.editGroupPage": "Na této stránce můžete upravit vlastnosti a členy skupiny. V horní části můžete upravit název a popis skupiny, pokud se nejedá o skupinu admin pro kolekci nebo komunitu. V takovém případě jsou název a popis skupiny vygenerovány automaticky a nelze je upravovat. V následujících částech můžete upravovat členství ve skupině. Další podrobnosti naleznete v části [the wiki](https://wiki.lyrasis.org/display/DSDOC7x/Create+or+manage+a+user+group).",
```

Comment:
> v části [the wiki](https://wiki.lyrasis.org/display/DSDOC7x/Create+or+manage+a+user+group)

`[the wiki]` plus nevim, jestli ten odkaz bude fungovat, a jestli je vubec rozumny ten odkaz v tomhle miste mit